app/forms/RegistrationForm.scala

```diff
@@ -34,11 +34,11 @@ import javax.inject.Inject
 
 class RegistrationForm@Inject()(applicationConfig: ApplicationConfig) {
 
-  private def maxLengthWithError(maxLength: Int, error: String = "error.maxLength"): Constraint[String] = Constraint[String]("constraint.maxLength", maxLength) { o =>
+  private def maxLengthWithError(maxLength: Int, error: String): Constraint[String] = Constraint[String]("constraint.maxLength", maxLength) { o =>
     if (o == null) Invalid(ValidationError(error, maxLength)) else if (o.length <= maxLength) Valid else Invalid(ValidationError(error, maxLength))
   }
 
-  private def nonEmptyTrimmer(error: String = "error.required"): Mapping[String] =
+  private def nonEmptyTrimmer(error: String): Mapping[String] =
```

Comment:
While not an error I am curious about why all these defaults have been removed.

Reply:
they give warnings for not being used, the defaults were defined in 2 places